### PR TITLE
fix: fixing the change value at select-dropdown

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/project/project-detail/project-member/modules/ProjectMemberUpdateModal.vue [hunk 1]
```diff
@@ -160,11 +160,11 @@ const handleConfirm = async () => {
     emit('confirm');
     state.proxyVisible = false;
 };
-const handleSelectRoleItems = (roleItems) => {
+const handleSelectRoleItems = (roleItems: string) => {
```
Comment (piggggggggy) [inline apps/web/src/services/project/project-detail/project-member/modules/ProjectMemberUpdateModal.vue:163]: Suggestion: How about `handleSelectRoleItem`?

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue [hunk 2]
```diff
@@ -299,11 +295,11 @@ const handleUpdateExternalUser = async (inputUserItems: InputItem[], isValid: bo
     selectedExternalUserItems.value = [...selectedExternalUserItems.value, _addedUserItem];
     externalItemsErrors.value = errors;
 };
-const handleSelectRoleItems = (roleItems: RoleMenuItem[]) => {
+const handleSelectRoleItem = (roleItems: string) => {
```
Comment (yuda110) [inline apps/web/src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue:298]: suggestion: How about `roleItem` or `roleItemName` or else? 🧐

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Pull request merged**